1-tolean.py

```diff
@@ -493,12 +496,40 @@ def check_opt(opt, timeout, bitwidth, hide_progress):
 
   return True # succeeded, did not time out
 
+def sanitize_name(name):
+  renamed = re.sub(r'[()~&>|^=]', '', name)
+  renamed = re.sub(r'[:, -]', '_', renamed)
+  return renamed
+
+def gather_names_widths(constants):
```

Comment:
I prefer to name such functions based on the key-value. So I would call it:
```suggestion
def build_width2constants(name2constants):
"""
build a map mapping each bitwidth to the list of constants
with that bitwidth. This is used when producing Lean code
to declare variables as `(a b c : Bitvec 1) (d e f : Bitvec 2)
"""
```